ee/modules/025-static-routing-manager/crds/routingtables.yaml

```diff
@@ -100,6 +100,8 @@ spec:
                   A set of labels that determine the nodes on which the desired routes should be created.
```

Comment:
```suggestion
                  A set of labels that determine the nodes on which the desired routes should be created.
                  
                  **Format**: the standard `nodeSelector` list.
```

Reply:
Let's simplify? We use this phrase in some other resources. [For example](https://deckhouse.io/products/kubernetes-platform/documentation/v1/modules/user-authn/cr.html#dexauthenticator-v1-spec-nodeselector)